chromium/preloadlist/preloadlist.go

```diff
@@ -54,12 +54,16 @@ type PreloadList struct {
 //
 // - Mode: The only valid non-empty value is ForceHTTPS
 //
-// - IncludeSubDomains: If Mode == ForceHTTPS, forces HSTS to apply to
-//   all subdomains.
+// - IncludeSubDomains: If Mode == ForceHTTPS, forces HSTS to apply to all subdomains.
```

Comment:
nit: don't change the formatting of these lines

Reply:
Sorry about that, I'm not actually changing any of these lines though haha. Whenever I save the document, it auto formats these lines for some reason but I'll try and fix that